packages/hardhat-astar-plugin/package-lock.json

```diff
@@ -0,0 +1,3471 @@
+{
```

Comment:
This file shouldn't be necessary afaik. Best to use yarn but overall should be contained in a master `yarn.lock` if the resolutions are properly set up.